### PR TITLE
Change ISSUE_TEMPLATE to make it easier to bootstrap an issue

### PR DESCRIPTION
**Pull request by ghost**
In order to be make it easier to bootstrap issues.

### CODE REVIEW

**Comment by dckc** [comment]
didn't we talk about good PR titles?

 - [Microcontent: A Few Small Words Have a Mega Impact on Business][1]

[1]: https://www.nngroup.com/articles/microcontent-how-to-write-headlines-page-titles-and-subject-lines/

**Comment by dckc** [comment]
more concise? I see a lot more lines added than subtracted.

**Comment by ICA3DaR5** [comment]
I'll edit this.

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,28 +1,65 @@
-# All
+# Issue
 
 _Please make the issue SMART: Specific, Measurable, Achievable, Realistic, Timely._
 
-Measure of Completion:  
-Suggested Budget:
+**Content:**
```
Comment (dckc) [inline ISSUE_TEMPLATE.md]: Please don't use **bold** markup where you mean a ...

### Header

**Review by dckc — DISMISSED** [review]

File: ISSUE_TEMPLATE.md (hunk 1 above)
Comment (ICA3DaR5) [reply ISSUE_TEMPLATE.md]: True.

**Pull request closed without merging**